client/overview/survey/index.tsx

```diff
@@ -0,0 +1,183 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { HorizontalRule } from '@wordpress/primitives';
+
+/**
+ * Internal dependencies.
+ */
+import {
+	Card,
+	CardBody,
+	Button,
+	TextareaControl,
+	Icon,
+} from '@wordpress/components';
```

Comment:
please move `@wordpress/components` to external dependencies

Reply:
fixed in https://github.com/Automattic/woocommerce-payments/pull/8309/commits/58b196fee6b384e1a266ea2a9c8841fbcad43cc6 . thx